lxd/loki/loki.go

```diff
@@ -232,11 +233,17 @@ func (c *Client) HandleEvent(event api.Event) {
 		return
 	}
 
+	hostname, err := os.Hostname()
+	if err != nil {
+		return
```

Comment:
Would it make sense to set the hostname to `none` in case of error?

Reply:
Absolutely.